transactions.go

```diff
@@ -0,0 +1,58 @@
+package breez
+
+import (
+	"context"
+	"github.com/breez/lightninglib/lnrpc"
+	"io"
+	"time"
+)
+
+func watchSwapTransactions() {
+	for {
+		err := subscribeTransactionsOnce()
+		if err != nil {
+			log.Errorf("Error trying to watchSwapTransactions:", err)
+		}
+		time.Sleep(5 * time.Second)
+	}
+}
+
+func subscribeTransactionsOnce() error {
+	transactionStream, err := lightningClient.SubscribeTransactions(context.Background(), &lnrpc.GetTransactionsRequest{})
```

Comment:
I think (although not sure) that SubscribeTransaction only gets new notifications so If the app was down we may loose transactions for this address